charmcraft.yaml

```
@@ -35,27 +35,40 @@ requires:
     optional: false
     limit: 1
 
+actions:
+  blacklist-and-delete-key:
+    description: Blacklist and delete a key fingerprint from the keyserver database.
+    properties:
+      fingerprints:
+        type: string
+        description: Comma-separated list of full fingerprints to ignore (minus the leading 0x)
+      ticket-id:
+        type: string
+        description: RT Ticket number corresponding to the GDPR request.
+    required: 
+      - fingerprints
+      - ticket-id
+  rebuild-prefix-tree:
+    description: The prefix tree is used to manage the reconciliation process and 
+      must always be in sync with the database. If you suspect it has become corrupted or out 
+      of sync, or if the prefix tree needs to be removed and rebuilt after pg-restore, 
+      it can be done with this action. 
```

```suggestion
    description: |
      The prefix tree is used to manage the reconciliation process and 
      must always be in sync with the database. If you suspect it has become corrupted or out 
      of sync, or if the prefix tree needs to be removed and rebuilt after pg-restore, 
      it can be done with this action. 
```
May I suggest use of either `>` or `|` to keep it multiline friendly? :) 
https://yaml-multiline.info/

---

Same for other yaml declarations :)

---

addressed